src/content/app/genome-browser/components/track-panel/components/track-panel-list/TrackPanelList.tsx

```diff
@@ -124,6 +124,7 @@ export const TrackPanelList = () => {
         >
           {selectedTrackPanelTab === TrackSet.VARIATION && (
             <TrackPanelVariantGroupLegend
+              shouldShowLegend={trackCategoryIds.length > 0}
               groups={variantGroups}
```

Comment:
Also missed this last time. Why are we passing this as a prop to `TrackPanelVariantGroupLegend` instead of importing it directly from `TrackPanelVariantGroupLegend`?